ethdb/database.go

```diff
@@ -207,12 +177,17 @@ func (db *LDBDatabase) Meter(prefix string) {
 //      1   |         85 |     109.27913 |      28.09293 |     213.92493 |     214.26294
 //      2   |        523 |    1000.37159 |       7.26059 |      66.86342 |      66.77884
 //      3   |        570 |    1113.18458 |       0.00000 |       0.00000 |       0.00000
+//
+// This is how the iostats look like (currently):
+// Read(MB):3895.04860 Write(MB):3654.64712
 func (db *LDBDatabase) meter(refresh time.Duration) {
 	// Create the counters to store current and previous values
 	counters := make([][]float64, 2)
 	for i := 0; i < 2; i++ {
 		counters[i] = make([]float64, 3)
 	}
+	// Create storage for iostats.
+	var curr, prev [2]float64
```

Comment:
You only need to track the previous values, there's no point to pre-declare a `curr` array too. Also previously we only gathered compaction stats, so names like `counters` was good enough, since there wasn't ambiguity. With the added iostats, `counters` and `curr` and `prev` all of a sudden don't mean anything.

Please rename `counters` to `compactions` and `curr/prev` to `iostats`.